### DISCUSSION TITLE
Gem mining tracker

### DISCUSSION THREAD

**Discussion by rvolkmar** [runelite/runelite#16433] [Ideas]
I'm planning on mining my way to 99 using gem rocks. It would be cool if there were a plugin to show me how many of each gem I've mined and the total profit/profit per hour I'm making along the way. I tried using the Profit Tracker plugin but it resets the profit every time you bank the gems.

**Comment by GuNeRkId** [comment]
For those who find this discussion and like the idea, I've made a basic version of this. Just search "gem tracker" in the plugin-hub.
